### DISCUSSION TITLE
Add docker host machine as node to teleport running in container

### DISCUSSION THREAD

**Discussion by lucacastelnuovo** [gravitational/teleport#14951] [Q&A] [answered]
Hi,

**Situation**
I have the teleport server running in a docker container in a VM. 
The programs `teleport`, `tctl`, `tsh` do not exist on the host.

**Goal**
I would like to ssh into the VM.

**What did I try**
I tried using the auto install script, the response was:
![image](https://user-images.githubusercontent.com/26206253/181304124-b1a5fce8-b216-4e26-9bc1-a9c334832109.png)

The files mentioned in this error do not exist on the host.

How do I solve this?
Thanks in advance

**Comment by lucacastelnuovo** [comment, marked as answer]
I fixed this by following this article: https://kifarunix.com/setup-secure-ssh-access-on-linux-servers-using-teleport/

**Reply by qeba** [reply]
What is the fix in short answer? How can I connect the host machine where the teleport is hosted on docker?..

**Reply by lucacastelnuovo** [reply]
Ah man, I don't know anymore. It's been a while ago. Just make sure you check the IPs and ports thoroughly.
